python/cudf/cudf/core/dataframe.py

```diff
@@ -1391,9 +1391,8 @@ def index(self, value):
         if isinstance(value, cudf.core.multiindex.MultiIndex):
             if len(self._data) > 0 and len(value) != len(self):
                 msg = (
-                    f"Length mismatch: Expected axis has "
-                    "%d elements, new values "
-                    "have %d elements" % (len(self), len(value))
+                    f"Length mismatch: Expected axis has {len(self)} elements,"
```

Comment:
Do these `f`s need to be removed? (other files as well).

Reply:
No, f-strings are a python thing that allow you to use the `{ }` notation to use python expressions inside of strings.

Reply:
Other changes in this same commit removed the f from strings in categorical.py and column.py. @karthikeyann please verify.

Reply:
Ah, I guess f was not required in those instances due to no `{ }` notation.

Reply:
Yes. f-strings without `{ }` are flagged as warning in flake8 `F541 f-string is missing placeholders`. So f was removed in those cases. 
This line has `{ }` placeholder. so, f is needed here.